packages/web-sdk/src/instrumentations/performance/navigation.ts

```diff
@@ -23,17 +26,38 @@ export function getNavigationTimings(
       return;
     }
 
+    const navEntryJson = navigationEntryRaw.toJSON();
+
+    let spanContext: Pick<SpanContext, 'traceId' | 'spanId'> = {
+      traceId: '',
+      spanId: '',
+    };
+
+    if ('serverTiming' in navEntryJson) {
```

Comment:
`PerformanceServerTiming` objects have n explicit structure. How did you receive an array instead?

Reply:
I think we can simmplify this function and be more expliciut. 

```suggestion
    const { serverTiming = [] }: { serverTiming: PerformanceServerTiming[] } = navEntryJson;

    for (const serverEntry of serverTiming) {
      if (serverEntry.name === 'traceparent') {
        const [traceId, spanId] = serverEntry.description.split('-');
        spanContext.traceId = traceId;
        spanContext.spanId = spanId;
        break;
      }
    }
```

Reply:
Does thge API contract guarantee that we always get traceID and spanID?

Reply:
Please add a test